pkgs/development/python-modules/tika-client/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pythonPackages
+}:
+buildPythonPackage rec {
+  pname = "tika-client";
+  version = "0.1.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "stumpylog";
+    repo = "tika-client";
+    rev = version;
+    hash = "sha256-c/4zoXxxrKA5uIz0esdNBmX1DYOiXrkCH1ePGUpEXRY=";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
```

No `pythonPackages` usage within `python-modules`, please. Please list all dependencies as function arguments.

```suggestion
  propagatedBuildInputs = [
```

---

Thanks, fixed it.